assets/dev/scss/editor/panel/controls/_media.scss

```diff
@@ -245,3 +245,34 @@
 		}
 	}
 }
+
+.elementor-control .elementor-control-content.elementor-control-media-size {
```

Comment:
try to put this CSS inside the existing one (use SASS concatenation things `&-size`, etc.)

Reply:
Changed.